Readability.js

```diff
@@ -1696,6 +1696,15 @@ Readability.prototype = {
     this._flags = this._flags & ~flag;
   },
 
+  _isRtl: function(str){           
+      var ltrChars    = 'A-Za-z\u00C0-\u00D6\u00D8-\u00F6\u00F8-\u02B8\u0300-\u0590\u0800-\u1FFF'+'\u2C00-\uFB1C\uFDFE-\uFE6F\uFEFD-\uFFFF',
+          rtlChars    = '\u0591-\u07FF\uFB1D-\uFDFD\uFE70-\uFEFC',
+          rtlDirCheck = new RegExp('^[^'+ltrChars+']*['+rtlChars+']');
```

Comment:
In fact, this regexp sees odd - the string should start with any number of non-LTR characters and then have 1 RTL character?